services/convert/user.go

```diff
@@ -75,6 +75,7 @@ func toUser(ctx context.Context, user *user_model.User, signed, authed bool) *ap
 	if authed {
 		result.IsAdmin = user.IsAdmin
 		result.LoginName = user.LoginName
+		result.SourceID = user.LoginSource
```

Comment:
I can see other fields are all 1-vs-1 mapping to user's field (eg: IsAdmin, LoginName, LastLogin, Language, IsActive, etc)

But why this field's name (SourceID) is different from the user's field (LoginSource)?

For consistency, I would prefer to use the same name.

Reply:
I agree with you, but as I have described in the PR, the use of the name _sourceID_ for the field is already being used in other places like
[Swagger CreateUserOption](https://github.com/go-gitea/gitea/blob/2033eb7c1138a06ea052e6c6e6e5799187519e16/templates/swagger/v1_json.tmpl#L19063)
[CreateUserOption struct](https://github.com/go-gitea/gitea/blob/2033eb7c1138a06ea052e6c6e6e5799187519e16/modules/structs/admin_user.go#L11)

I could change to *result.LoginSource = user.LoginSource* and then map that to the confusing, but used for consistency, _source_id_ in the swagger file? I guess that would be better.

Let me know what you think. Thanks.

Reply:
Hmm ... good point, that's a legacy problem .... so it's better to keep using "SourceID"